harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -749,23 +589,43 @@ class PyTorchTrial(det.Trial):
     trial_controller_class = PyTorchTrialController
     trial_context_class = PyTorchTrialContext
 
-    @abstractmethod
+    def __init__(self, trial_context: PyTorchTrialContext) -> None:
```

Comment:
blocking: we should not have a default `__init__()`.  Users *must* define their own.